zebra-rpc/src/methods.rs

```diff
@@ -733,200 +746,212 @@ where
         hash_or_height: String,
         verbosity: Option<u8>,
     ) -> BoxFuture<Result<GetBlock>> {
-        // From <https://zcash.github.io/rpc/getblock.html>
-        const DEFAULT_GETBLOCK_VERBOSITY: u8 = 1;
-
         let mut state = self.state.clone();
-        let verbosity = verbosity.unwrap_or(DEFAULT_GETBLOCK_VERBOSITY);
 
         async move {
-            let hash_or_height: HashOrHeight = hash_or_height.parse().map_server_error()?;
```

Comment:
These changes to the `get_block()` method may conflict with those in https://github.com/ZcashFoundation/zebra/pull/9006.

Reply:
Let's resolve those when we merge either of the PRs.